types/mssql/mssql-tests.ts

```diff
@@ -96,16 +96,16 @@ var connection: sql.ConnectionPool = new sql.ConnectionPool(config, function (er
         var testString: string = 'test';
 
         requestStoredProcedureWithOutput.input("name", sql.VarChar, "abc");               // varchar(3)
-        requestStoredProcedureWithOutput.input("name", sql.VarChar(50), "abc");           // varchar(MAX)
+        requestStoredProcedureWithOutput.replaceInput("name", sql.VarChar(50), "abc");           // varchar(MAX)
         requestStoredProcedureWithOutput.output("name", sql.VarChar);                     // varchar(8000)
         requestStoredProcedureWithOutput.output("name", sql.VarChar, "abc");              // varchar(3)
 
-        requestStoredProcedureWithOutput.input("name", sql.Decimal, 155.33);              // decimal(18, 0)
-        requestStoredProcedureWithOutput.input("name", sql.Decimal(10), 155.33);          // decimal(10, 0)
-        requestStoredProcedureWithOutput.input("name", sql.Decimal(10, 2), 155.33);       // decimal(10, 2)
+        requestStoredProcedureWithOutput.replaceInput("name", sql.Decimal, 155.33);              // decimal(18, 0)
+        requestStoredProcedureWithOutput.replaceInput("name", sql.Decimal(10), 155.33);          // decimal(10, 0)
+        requestStoredProcedureWithOutput.replaceInput("name", sql.Decimal(10, 2), 155.33);       // decimal(10, 2)
```

Comment:
Ideally, the test should cover all cases for both `input` and `replaceInput`.
So please copy-paste instead of overwriting (I cannot beleive that I just asked someone to copy-paste code 😄)

Also, it seems that neither of them have a test for the case without a type argument, e.g. 

```typescript
requestStoredProcedureWithOutput.input("name", "abc");
requestStoredProcedureWithOutput.replaceInput("name", "abc");
```